.github/workflows/debezium-workflow.yml

```diff
@@ -291,11 +291,10 @@ jobs:
           -Dmaven.wagon.http.pool=false
           -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
 
-  ## Add this as until 5.7 will be EOL in October 2023
-  build_mysql_5_7:
+  build_mysql_8_0:
```

Comment:
I think you should completely remove this step since the one for MySQL 8.0 (current) is the `build_mysql`

Reply:
@mfvitale The current `build_mysql` is based on MySQL 8.2 now.

Reply:
Oh ok. So now we want to test 8.0 and 8.2?

Reply:
Yes, right.

Reply:
@mfvitale Just FYI, MySQL 8.0 is LTS, so that's the reason for choosing it as baseline.

Reply:
Got it. Thanks.